sdks/python/apache_beam/io/filesystems.py

```diff
@@ -218,6 +218,25 @@ def exists(path):
     filesystem = FileSystems.get_filesystem(path)
     return filesystem.exists(path)
 
+  @staticmethod
+  def checksum(path):
+    """Fetch checksum metadata of a file on the FileSystem.
+
+    This operation returns checksum metadata as stored in the underlying
+    FileSystem. It should not read any file data. Checksum type and format are
```

Comment:
<!--new_thread; commit:24c81c1018b47c5316f09f3b1eef8725b5492848; resolved:0-->
Ditto (doc comment formatting for classes).